Dockerfile

```diff
@@ -1,7 +1,13 @@
 from node
 RUN mkdir /app
 WORKDIR /app
-COPY [ "package.json", "src", "public", "/app/"]
-RUN yarn
-EXPOSE  3000
+ENV PORT=80
+ENV NODE_PATH=src
+COPY yarn.lock /app/yarn.lock
+COPY .yarn-cache /.yarn-cache
+COPY package.json /app/package.json
+COPY public /app/public
+COPY src /app/src
+RUN yarn install --cache-folder /.yarn-cache
```

Comment:
This should go immediately after copying `yarn.lock` & `package.json`. That way we'll only reinstall when those files change. Currently the dependencies will be reinstalled if any of the `src` or `public` files change 🙀